lab-raziyeh/data/book/e6f858d0-84f8-11e6-b67b-3bf571033bdf.json

```diff
@@ -0,0 +1 @@
+{"id":"e6f858d0-84f8-11e6-b67b-3bf571033bdf","title":"JavaScript-2","author":"Raziyeh Bazargan","page":"120","creationDate":"2016-09-27T21:25:24.317Z"}
```

Comment:
@Raziyehbazargan your data directory can be added to your gitignore file in order to keep these from being directly pushed to the repo - this is simply to keep things organized and clean.